tests/Microsoft.Identity.Test.Unit/ManagedIdentityTests/PersistentCertificateStoreUnitTests.cs

```diff
@@ -0,0 +1,553 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
```

Comment:
Use private helper methods to reduce repeated code for Windows checks or lock calls

Reply:
done